autoload/vimlparser.vim

```diff
@@ -1121,6 +1121,18 @@ function! s:VimLParser.parse_cmd_common() abort
         break
       endif
     endwhile
+  elseif self.ea.cmd.name ==# 'def'
+    " TODO: support vim9script
+    while !self.reader.eof()
+      if self.reader.peek() ==# '#'
+        call self.reader.readline()
+      endif
```

Comment:
This should be "else" ? After called readline, seek pos might be already EOF before enddef.